### DISCUSSION TITLE
Welcome to ML-For-Beginners Discussions!

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jlooper** [comment]
My name is Jen, I'm the lead on this project and it's my pleasure to present this, and other curricula, from the Education Cloud Advocacy Team at Microsoft. We hope you learn a lot from these lessons. You can read about me here: [aka.ms/jenlooper](https://aka.ms/jenlooper) or visit me on the web at [jenlooper.com](https://jenlooper.com) ❤️

**Reply by rupen4678** [reply]
okey am soory for stupid question but how do i start the project i run the index.html file but nothing shows up should i goto each and every folder

**Reply by jlooper** [reply]
hi @rupen4678 what are you trying to run? These lessons are contained as markdown files in folders, with notebooks showing the solutions in their respective folders

**Reply by rupen4678** [reply]
@jlooper yes but it dosent run it says it dosent have json format or so

**Reply by jlooper** [reply]
hi @rupen4678 can you tell me what file you're trying to run and the error you're getting?

**Reply by rupen4678** [reply]
i tried to run the index.html file i saw the aource it had nothing so i went to the folder and opened the .ipynb files

**Reply by jlooper** [reply]
Please read all the instructions on how to run the repo locally on the home README file. You need to use Docsify to run the web version of this curriculum locally, and you will be working in the empty .ipynb files to build something like what's in the solution folder for each lesson.

**Reply by AkshatRaj00** [reply]
Hey everyone 👋

I'm Akshat Raj, a Computer Engineering student passionate about AI, Machine Learning, and Open Source development. I'm currently working on projects like emotion detectors, AI assistants, and robotics using Python + React.

I'm here to learn, contribute to meaningful discussions, and collaborate on exciting projects. Looking forward to engaging with this amazing community and sharing knowledge 🙌

Let’s build and grow together 🚀

**Reply by ravi19ved** [reply]
Hi Everyone, 

I am Ravi Chandra, A professional with physics background and now exploring Data Science and ML for using in potential scientific application. Happy to connect with all the enthusiastic community around. 

Lets learn and build great stuff together. 

Cheers.

**Comment by ghost** [comment]
Hello, my name is Tofig Nazarbayov! I am master student of Geophysical Engineering via dual diploma program at University of Strasbourg in France and Azerbaijan State Oil and Industry University in Azerbaiajn. I am passionate about machine learning, and am happy to contribute to this project!

You can find more professional info about me at: linkedin.com/in/tofignazarbayov

**Reply by jlooper** [reply]
Bienvenue Tofig! Happy to have you here

**Comment by rohith-pandala** [comment]
On which date this course is begin....

**Reply by jlooper** [reply]
hi, this is a self-study course, so begin when you like!

**Comment by nekomar** [comment]
Hi! My name is Omar, and I'm from Azerbaijan as well, and would like to switch from civil engineering to data science/ML industry. I hope your course will help me to get started!

**Reply by jlooper** [reply]
Welcome! I hope so as well 👋

**Comment by amlan01** [comment]
Hello, my name is Amlan. I'm from India. Currently working with the state remote sensing center in Assam, India. I hope this community will help augment my ML learning journey. Moreover, i intend to build feature extraction models for satellite data. I have come across U-Net models, yet never deployed one to work. Pinning my hopes on this space to help understand the fundamentals. Thank you. Reach me @[Linkedin](https://in.linkedin.com/in/amlansaikia)

**Reply by jlooper** [reply]
Welcome, Amlan!

**Comment by rabichubu** [comment]
My name is Rabi Mustapha. I want to a have a basic knowledge and understanding of Machine Learning.

**Reply by shmuhammadd** [reply]
@rabichubu Hi Rabi,  this is a self-study curriculum. You can start any time you wish. However, our online group study session ( Nigeria - one you sign-up your name ) will start next week. Keep an eye, you will receive an email within the next week.

**Reply by shmuhammadd** [reply]
@rabichubu  You can also join the Discord to interact with other learners. See #276

**Reply by rabichubu** [reply]
جذاكلله خيرا يا شمسدين

**Comment by rabichubu** [comment]
Thanks Mr Shamsuddeen

**Comment by rupesh-dharme** [comment]
For the assignment "interview a data scientist" in techniques of ML, we are asked to submit an essay. So how are we supposed to submit that?

**Reply by jlooper** [reply]
Since it’s a self study course you might submit it here for feedback if you like

**Comment by NanaPokun** [comment]
Hello, I am Felix, a Ghanaian and a lover of everything python..im a freelance self taught developer passionate about ML. Ready to learn and happy to be here.

**Reply by jsacevedor** [reply]
Hi Felix, I'm a beginner in this world, I'm interested in freelancer option, I asked you if maybe you would answer some questions to help me, please

**Comment by demboydeb** [comment]
Hello My name is Dema and I love ML. I hope that we will build the future together.

**Comment by erikweibust** [comment]
Howdy,

Thanks for putting this together. I have no ML experience nor do I have a background in Data Science. Would it make more sense to start with https://github.com/microsoft/Data-Science-For-Beginners and then come back to this course?

Thanks,
Erik

**Reply by jlooper** [reply]
hi! Great question! This course is for folks who are specifically interested in classical ML topics. I think the two courses standalone, and can be taken independently. The DS course is more about data wrangling, visualization, and using Azure, whereas this one leans heavily on Scikit-Learn. Either way, I'd encourage you to take both! Let us know how it goes!

**Reply by mubarak-mustopha** [reply]
I've been posting some of the assignments from the tutorial on my github https://github.com/mubarak-mustopha/microsoft_mlcourse
can you check them and give me a feedback please.

**Reply by erikweibust** [reply]
@jlooper Thanks for the quick, helpful reply. I'll take both! But, will probably start with the DS course and then the ML couse.

**Comment by abdull6771** [comment]
I'm begginer in Machine Learning and Computer Engineering Student from Nigeria I find this course very useful thank you Ma and I look forward to connect with you thank you once again

**Comment by pch8429** [comment]
Hello, my name is Chul (social scientist). First of all, thank you so much for such great content! I have a question about "[Build a regression model using Scikit-learn: prepare and visualize data](https://github.com/microsoft/ML-For-Beginners/blob/main/2-Regression/2-Data/README.md). I'm stuck in the following codes:


import pandas as pd
pumpkins = pd.read_csv('../data/US-pumpkins.csv')
pumpkins.head()

Whenever I ran these codes, I got the following messages. Please help me address this problem. Thank you in advance.

---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
/Users/chulpark/notebook.ipynb Cell [1](vscode-notebook-cell:/Users/chulpark/notebook.ipynb#ch0000000?line=0)' in <module>
      1[ import pandas as pd
----> ]()[2](vscode-notebook-cell:/Users/chulpark/notebook.ipynb#ch0000000?line=1)[ pumpkins = pd.read_csv('../data/US-pumpkins.csv')
      ]()[3](vscode-notebook-cell:/Users/chulpark/notebook.ipynb#ch0000000?line=2)[ pumpkins.head()

File /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py:311, in deprecate_nonkeyword_arguments.<locals>.decorate.<locals>.wrapper(*args, **kwargs)
    ]()[305](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=304)[ if len(args) > num_allow_args:
    ]()[306](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=305)[     warnings.warn(
    ]()[307](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=306)[         msg.format(arguments=arguments),
    ]()[308](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=307)[         FutureWarning,
    ]()[309](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=308)[         stacklevel=stacklevel,
    ]()[310](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=309)[     )
--> ]()[311](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/util/_decorators.py?line=310)[ return func(*args, **kwargs)

File /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py:680, in read_csv(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, squeeze, prefix, mangle_dupe_cols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, encoding_errors, dialect, error_bad_lines, warn_bad_lines, on_bad_lines, delim_whitespace, low_memory, memory_map, float_precision, storage_options)
    ]()[665](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=664)[ kwds_defaults = _refine_defaults_read(
    ]()[666](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=665)[     dialect,
    ]()[667](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=666)[     delimiter,
   (...)
    ]()[676](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=675)[     defaults={"delimiter": ","},
    ]()[677](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=676)[ )
    ]()[678](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=677)[ kwds.update(kwds_defaults)
--> ]()[680](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=679)[ return _read(filepath_or_buffer, kwds)

File /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py:575, in _read(filepath_or_buffer, kwds)
    ]()[572](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=571)[ _validate_names(kwds.get("names", None))
    ]()[574](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=573)[ # Create the parser.
--> ]()[575](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=574)[ parser = TextFileReader(filepath_or_buffer, **kwds)
    ]()[577](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=576)[ if chunksize or iterator:
    ]()[578](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=577)[     return parser

File /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py:933, in TextFileReader.__init__(self, f, engine, **kwds)
    ]()[930](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=929)[     self.options["has_index_names"] = kwds["has_index_names"]
    ]()[932](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=931)[ self.handles: IOHandles | None = None
--> ]()[933](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=932)[ self._engine = self._make_engine(f, self.engine)

File /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py:1217, in TextFileReader._make_engine(self, f, engine)
   ]()[1213](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1212)[     mode = "rb"
   ]()[1214](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1213)[ # error: No overload variant of "get_handle" matches argument types
   ]()[1215](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1214)[ # "Union[str, PathLike[str], ReadCsvBuffer[bytes], ReadCsvBuffer[str]]"
   ]()[1216](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1215)[ # , "str", "bool", "Any", "Any", "Any", "Any", "Any"
-> ]()[1217](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1216)[ self.handles = get_handle(  # type: ignore[call-overload]
   ]()[1218](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1217)[     f,
   ]()[1219](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1218)[     mode,
   ]()[1220](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1219)[     encoding=self.options.get("encoding", None),
   ]()[1221](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1220)[     compression=self.options.get("compression", None),
   ]()[1222](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1221)[     memory_map=self.options.get("memory_map", False),
   ]()[1223](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1222)[     is_text=is_text,
   ]()[1224](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1223)[     errors=self.options.get("encoding_errors", "strict"),
   ]()[1225](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1224)[     storage_options=self.options.get("storage_options", None),
   ]()[1226](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1225)[ )
   ]()[1227](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1226)[ assert self.handles is not None
   ]()[1228](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/parsers/readers.py?line=1227)[ f = self.handles.handle

File /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py:789, in get_handle(path_or_buf, mode, encoding, compression, memory_map, is_text, errors, storage_options)
    ]()[784](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=783)[ elif isinstance(handle, str):
    ]()[785](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=784)[     # Check whether the filename is to be opened in binary mode.
    ]()[786](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=785)[     # Binary mode does not support 'encoding' and 'newline'.
    ]()[787](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=786)[     if ioargs.encoding and "b" not in ioargs.mode:
    ]()[788](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=787)[         # Encoding
--> ]()[789](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=788)[         handle = open(
    ]()[790](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=789)[             handle,
    ]()[791](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=790)[             ioargs.mode,
    ]()[792](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=791)[             encoding=ioargs.encoding,
    ]()[793](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=792)[             errors=errors,
    ]()[794](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=793)[             newline="",
    ]()[795](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=794)[         )
    ]()[796](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=795)[     else:
    ]()[797](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=796)[         # Binary mode
    ]()[798](file:///Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pandas/io/common.py?line=797)[         handle = open(handle, ioargs.mode)

FileNotFoundError: [Errno 2] No such file or directory: '../data/US-pumpkins.csv']()

**Reply by jlooper** [reply]
hi! the issue looks like a missing file or a misconfigured path. This error: FileNotFoundError: [Errno 2] No such file or directory: '../data/US-pumpkins.csv']() shows that your notebook can't find the pumpkin data. Check to see where that data file exists and make sure your path is well configured. Then it should run ok!

**Reply by pch8429** [reply]
Thank you!

**Comment by pch8429** [comment]
Hello, I wanted to clone "https://github.com/microsoft/ML-For-Beginners.git" on my Terminal, following [Alfredo Deza's guidance](https://www.youtube.com/watch?v=yyQM70vi7V8). But it is not working with the following error message. Is there anyone who helps me resolve this issue? Thank you in advance.

(base) ITs-MBP-2:~ chulpark$ git clone https://github.com/microsoft/ML-For-Beginners.git
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
(base) ITs-MBP-2:~ chulpark$

**Reply by jlooper** [reply]
hi, sounds like this problem: https://developer.apple.com/forums/thread/673827. Maybe you need to do a reinstallation of your command line tools?

**Reply by pch8429** [reply]
I will take a look at this. Thank you so much!

**Reply by pch8429** [reply]
You're right. It worked!

**Reply by jlooper** [reply]
yay!

**Comment by king-burna19** [comment]
Hello my name is Barnabas a studen from Kenya, doing Mechatronic Engineering. I was wondering how do i start this course??

**Reply by AkashahS** [reply]
start by forking the repo to your account. then go to your account and open the forked repository "ML-For-Beginners" open folders "1. Introduction" and then 1."1-intro-to-ML" and follow along.

**Comment by dwonisone** [comment]
You should have let the Romans take u instead of switching with jesus

**Comment by ProducerTonics** [comment]
hii im kelvin and my problem or suggestion is i need help on how to use or create html

**Comment by Samuel-Batubo** [comment]
Good day Everyone, I'm Samuel a post graduate student of Gas Engineering and a passionate ML enthusiast. I will love to learn with everyone here.

**Comment by Nyalwendo** [comment]
Hell, My name is James Omina, I am looking forward to learning more about Machine Learning (ML) on this planform, as i am eager to putting it into the real life practice,

**Comment by peterkahumu** [comment]
Hi everyone, my name is Peter, a computer science student at the Multimedia University of Kenya. I hope to learn more and gain hands-on skills in the field of Artificial Intelligence.

**Comment by dwonisone** [comment]
Yes how are you can u help me with this mimic problem I have...some one is
mimicing me and I need the info switched back to my accuse info

**Comment by sajjadrahman56** [comment]
Hello ML enthusiasts 

I am Sajjad Rahman from Bangladesh. 
Just started my ML  journey.  Hope the more we will learn together .

**Reply by Sanjaypranav** [reply]
all  the best sajjad-njr

**Comment by Bmoyer96** [comment]
Greetings, 
     My name is 
B. Allen Moyer, I am honored to be here and hope I can in some form of fashion accomplish each of you to some meaningful capacity. For quite a few years now I had wasted my value on they who not only took it but also went along to make many millions in monetary profits with my often times simple advice. Perhaps one of you is able to teach me how to accrue some of that wealth back? 

Sincere gratitude, Benji

**Comment by Divya-Kumari-Gupta** [comment]
hello ma'am I am Divya .i want to learn ML but i dont where to stated

**Comment by Falco361** [comment]
I would go to microsoft website  and search for there learning sections. Also. Looking into I.T. Courses. And a lot can be found on YouTube ,it’s what helped me and I was guided to southern career institute   Where I’ve just started in it .ide look into Microsoft websites. On there learning sections might be the best option. Tho , best of luck to you on your journey,

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ahmedosama990** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

I just want to ask how can I learn this repo with the data science repo in a correct and meaningful way

**Comment by Bmoyer96** [comment]
Greetings guys & gals! I have a couple stellar ideas for bots but I have a tough time buckling down and making myself learn to code. I don’t have anyone that I know of who does it too well either sadly. Any advice?

**Comment by rosh93an** [comment]
hey hi , i am new to ml i need to learn time series forecasting can some one he;p with it i have a project to submit
[energy_dataset.csv](https://github.com/user-attachments/files/17311935/energy_dataset.csv)
[weather_features.csv](https://github.com/user-attachments/files/17311944/weather_features.csv)
approach for forecasting electricity prices based on the hourly energy demand, energy generation from various sources, and the weather parameters given in the attached dataset.

**Comment by pattaneja** [comment]
Hello! I’m Prateek Taneja from India.

With 10 years of international experience in SaaS platforms, specializing in ERP, CRM, and sales, I bring a strong foundation in both business and technology. Driven by a growing passion for development, I’ve started learning Salesforce development and am now eager to contribute to open source projects.

I’m seeking guidance and mentorship on how best to get started in this area — especially in the Salesforce ecosystem. Any advice or pointers would be highly appreciated!

**Reply by AkshatRaj00** [reply]
Hello,can we talk

**Reply by pattaneja** [reply]
yes

**Comment by Lex-online** [comment]
Hello,

I am Lex. After losing my job, I am looking into moving to a new industry and I have always been interested in tech / coding.

**Comment by AungSaw-30** [comment]
Hello Jen love to see you and I wanna learn about ML
